server/app.js

```diff
@@ -26,7 +26,12 @@ app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
 
 if (process.env.ASSET_COMPUTE_DEV_TOOL_ENV !== 'development') {
-    app.use(express.static(path.join(__dirname, 'client-build')));
+    if (process.platform === "darwin" || process.platform === "linux") {
+        app.use(express.static(path.join(__dirname, 'client-build')));
+    }
+    else if (process.platform === "win32") {
+        app.use(express.static(path.join(__dirname, '\\client-build')));
```

Comment:
Join is suppoed to take platforms into account when joining: (from Nodejs doc): 
`The path.join() method joins all given path segments together using the platform-specific separator as a delimiter, then normalizes the resulting path.`.

Please add a unit test for that behavior and use Travis' build capabilities for Linux and WIndows (and log the resulting joined paths). I'd like to see if platform filtering is really needed here (as long as we can avoid it, we should).